.gitignore

```diff
@@ -12,3 +12,5 @@ dist
 htmlcov
 venv
 *.sw[op]
+
+.idea/
```

Comment:
does this have to be here?

Reply:
Unless you want commiting my .idea dir contents

Reply:
doesn't seem like it should exist in the repo though

Reply:
IntelliJ always puts it's .idea directory in the repos.